frontend/src/components/ProfileSection/ProfileSection.test.tsx

```diff
@@ -2,28 +2,22 @@ import { render, screen } from "@testing-library/react";
 import "@testing-library/jest-dom";
 import { faker } from "@faker-js/faker";
 
-import ProfileSection from ".";
+import ProfileDetails from ".";
```

Comment:
would it be better to standardise the name for the ProfileSection and ProfileSection.test to ProfileDetails as well?